modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v1_1_3/DDMStorageLinkUpgradeProcess.java

```diff
@@ -104,6 +119,7 @@ protected void doUpgrade() throws Exception {
 			}
 
 			preparedStatement4.executeBatch();
+			preparedStatement3.executeBatch();
```

Comment:
@vngoliveira please follow the alphabetical order, `preparedStatement3` before `preparedStatement4`